.eslintrc.yml

```diff
@@ -0,0 +1,11 @@
+env:
+  browser: true
```

Comment:
Not a broswer tool

Reply:
`eslint --init` initialized that like that (after asking if it was browser-based or node-based)
I'll remove it and check if it is still passing